modules/nw-sriov-dual-nic-proc.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-sriov-device.adoc
+
+:_content-type: CONCEPT
+[id="nw-dual-nic-proc_{context}"]
+= Setting up networking for dual-nic on day-1
+
+== Agent based installer
+
+The agent-based installer applies any network configuration that is in NMState format. Network configuration is defined in the `agent.config.yaml` file
+
+.Example: Bonds and SR-IOV dual-nic node network configuration
+This is a example YAML configuration file that defines a host in a cluster. The host has several network interfaces and a bond interface.
+
+.Sample agent-config.yaml.file
++
+[source,yaml]
+----
+[...]
+- hostname: worker-1 <1>
+    interfaces:      <2>
```

Comment:
Apply the same suggestions as mentioned in modules/agent-install-sample-config-bond-sriov.adoc

Reply:
Moved the content of this to installation section